<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeEditDock</class>
 <widget class="QDockWidget" name="NodeEditDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&amp;Node Edit</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout">
      <item>
       <widget class="QToolButton" name="enable_autoupdate">
        <property name="text">
         <string>Enable autoupdate</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="rainynite::studio::NodeStringEditor" name="text_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="custom_placeholder" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rainynite::studio::NodeStringEditor</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/node_string_editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
